changelogs/unreleased/gh-7149-forbid-ddl-until-box-schema-upgrade.md

```diff
@@ -0,0 +1,4 @@
+## feature/core
+
+- Disabled DDL operations with an old system schema. Now you have to run
+  `box.schema.upgrade()` before you can execute any DDL operations (gh-7149).
```

Comment:
It is a breaking change.

Reply:
You're right - my bad. Fix is here: https://github.com/tarantool/tarantool/pull/8708